### PR TITLE
install make command

### PR DESCRIPTION
**Pull request by Manali-Jain-squareops**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by cnlangzi — APPROVED** [review]
LGTM

File: .github/workflows/build-sdks.yml [hunk 1]
```diff
@@ -268,6 +271,8 @@ jobs:
         with:
           go-version: 1.19.4
 
+      - name: Install make
+        run: sudo apt update && sudo apt upgrade && sudo apt install build-essential -y
```
Comment (cnlangzi) [inline .github/workflows/build-sdks.yml]:
```suggestion
run: sudo apt update -y && sudo apt upgrade -y && sudo apt install build-essential -y
```

File: .github/workflows/build-sdks.yml [hunk 2]
```diff
@@ -119,6 +119,9 @@ jobs:
         id: setup-ndk
         with:
           ndk-version: r21d
+      
+      - name: Install make
+        run: sudo apt update && sudo apt upgrade && sudo apt install build-essential -y
```
Comment (cnlangzi) [inline .github/workflows/build-sdks.yml]:
```suggestion
run: sudo apt update -y && sudo apt upgrade -y && sudo apt install build-essential -y
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -41,6 +41,9 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
+      
+      - name: Install make
+        run: sudo apt update && sudo apt upgrade && sudo apt install build-essential -y
```
Comment (cnlangzi) [inline .github/workflows/tests.yml]:
```suggestion
run: sudo apt update -y && sudo apt upgrade -y && sudo apt install build-essential -y
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -139,6 +142,9 @@ jobs:
 
       - name: Setup PATH for wasm
         run: echo "${{env.GOROOT}}/misc/wasm" >> $GITHUB_PATH
+      
+      - name: Install make
+        run: sudo apt update && sudo apt upgrade && sudo apt install build-essential -y
```
Comment (cnlangzi) [inline .github/workflows/tests.yml]:
```suggestion
run: sudo apt update -y && sudo apt upgrade -y && sudo apt install build-essential -y
```

**Review by cnlangzi — DISMISSED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by cnlangzi** [comment]
@peterlimg  the fix doesn't work.

**Comment by cnlangzi** [comment]
fixed it on https://github.com/0chain/gosdk/pull/722

**Pull request merged**